emukit/core/loop/model_updaters.py

```diff
@@ -24,19 +25,22 @@ def update(self, loop_state: LoopState) -> None:
 
 class FixedIntervalUpdater(ModelUpdater):
     """ Updates hyper-parameters every nth iteration, where n is defined by the user """
-    def __init__(self, model: IModel, interval: int = 1) -> None:
+    def __init__(self, model: IModel, interval: int = 1, target_name='Y') -> None:
```

Comment:
I suggest a lambda to get training data out of loop state instead of this reflection thingy. Something like `lambda loop_state: loop_state.Y`

Reply:
Done